### PR TITLE
refactor: Clean unused code

### PR DESCRIPTION
**Pull request by caiopiccirillo**

### CODE REVIEW

**Comment by caiopiccirillo** [comment]
@LLagu Hey man, this might fell off your radar. Could you check for me when your have some time, please? 😄

**Comment by LLagu** [comment]
> @LLagu Hey man, this might fell off your radar. Could you check for me when your have some time, please? 😄

Hey man, sorry! I remember looking at this but I forgot to approve, my bad! 😅

**Review by LLagu — APPROVED** [review]

**Comment by caiopiccirillo** [comment]
> > @LLagu Hey man, this might fell off your radar. Could you check for me when your have some time, please? 😄
> 
> Hey man, sorry! I remember looking at this but I forgot to approve, my bad! 😅

No problem at all :smile:

File: src/morph_viewer_plugin.rs
[the remark this answers is not in the document]
Comment (caiopiccirillo) [reply src/morph_viewer_plugin.rs:1]: I decided to remove unused code to make it cleaner. If we need (and we will) those shortcuts, it'll be reintroduced.

**Pull request merged**